src/bundle/Resources/public/js/scripts/fieldType/base/base-rich-text.js

```diff
@@ -43,6 +43,7 @@
                     };
                 }
             );
+            this.editorPlugins = global.eZ.adminUiConfig.editorPlugins;
```

Comment:
The `editorPlugins` key is very generic, I would change this to `richtTextEditorPlugins`.